### DISCUSSION TITLE
Option to hide the disclaimer / info footer

### DISCUSSION THREAD

**Discussion by JJWatMyself** [f1multiviewer/issue-tracker#48] [Feature requests] [answered]
This is great information and UI would benefit from option to hide. Especially helpful on the Live Timing screen.

![image](https://user-images.githubusercontent.com/22590445/174445278-b8b3eb66-a740-46e4-a859-49ffc74a4ef4.png)

**Comment by f1multiviewer** [comment, marked as answer]
Done! See the settings window:

![image](https://user-images.githubusercontent.com/106821994/176937433-759457b2-f546-48ba-b430-cc97369e6fdf.png)

**Reply by JJWatMyself** [reply]
Awesome and thank you!
